routes/users.js

```diff
@@ -21,6 +21,13 @@ router.get("/:userId/intro", authenticate, authorizeRoles([SUPERUSER]), users.ge
 router.put("/self/intro", authenticate, userValidator.validateJoinData, users.addUserIntro);
 router.get("/:id/skills", users.getUserSkills);
 router.get("/:id/badges", getUserBadges);
+router.patch(
+  "/:userId/roles",
```

Comment:
should we call the path param `:id` to stay consistent with the previous lines

Reply:
If you see there is also used :userId in routes/user.js file -> https://github.com/Real-Dev-Squad/website-backend/blob/develop/routes/users.js#L20

Reply:
Done please check in latest changes